recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -40,8 +40,8 @@ def _source_subfolder(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "15",
-            "gcc": "5",
+            "Visual Studio": "15" if tools.Version(self.version) < "0.4.16" else "16",
+            "gcc": "9",
```

Comment:
This broke my builds (we use gcc 8).
Why do you think we need gcc 9 to build with C++14? These docs say 5 is enough.
https://gcc.gnu.org/projects/cxx-status.html#cxx14

Reply:
Depends the feature ... for this case, it seems like gcc 5 has what we need. We can drop to 8, for sure.